modules/plot/test/spec.coffee

```diff
@@ -1,8 +1,12 @@
+# Used to mimic an event call for a node
+fakeNodeEvent = (node, eventName) ->
+  if node?
+    (e) -> hx.select.getHexagonElementDataObject(node).eventEmitter?.emit((if eventName? and isNaN(eventName) then eventName else 'click'), e)
```

Comment:
We currently don't have a 'test helpers' file but I agree that one would be very useful - there are several things that we could put in there (e.g. emulating selecting a menu item etc)